assets/src/edit-story/app/api/apiProvider.js

```diff
@@ -142,18 +143,20 @@ function APIProvider({ children }) {
               featured_media: posterId,
               featured_media_src: poster,
               alt_text: alt,
-            }) => ({
-              id,
-              posterId,
-              poster,
-              src,
-              oWidth,
-              oHeight,
-              mimeType,
-              lengthFormatted,
-              alt: alt ? alt : description,
-              title,
-            })
+            }) =>
+              getAttachmentFromResource({
```

Comment:
I don't believe we would be using `getAttachmentFromResource(...)` here and with the `getAttachmentFromResource(getResourceFromFile(...))`. The object returned by `apiFetch` are not in the `Resource` format.